pandas/_libs/tslibs/offsets.pyx

```diff
@@ -243,10 +243,10 @@ def _validate_business_time(t_input):
 
 relativedelta_kwds = set([
     'years', 'months', 'weeks', 'days',
-    'year', 'month', 'week', 'day', 'weekday',
```

Comment:
technically this is an api change, IOW week & milliseconds cannot be passed anymore (maybe add this PR number somwhere)

Reply:
Shoot you're right.